serverlessworkflow/modules/ROOT/pages/cloud/kubernetes-service-discovery.adoc

```diff
@@ -30,6 +30,9 @@ The following scheme values are supported in the URI pattern:
 * `openshift`
 * `knative`
 
+> **NOTE**: When using `knative`, you can also use a simplified URI like:
```

Comment:
on a second thought, I would have put that one after the listing of resources and mentioning it is looking directly for `serving.knative.dev/v1/service` resource

Reply:
Done.
Also, I removed the warning about GVK being mandatory.